rocAL/rocAL/source/pipeline/node.cpp

```diff
@@ -70,3 +78,25 @@ Node::update_parameters()
 {
     update_node();
 }
+
+// Create vx_tensor for the crop coordinates
+void Node::create_crop_tensor(vx_tensor &crop_tensor, void **crop_coordinates) {
+    vx_size num_of_dims = 2;
```

Comment:
Why have we harded to 2 here?

Reply:
Since it is always 2 dimensional we have hardcoded to 2